main.go

```diff
@@ -10,14 +11,17 @@ import (
 	"os"
 	"path"
 	"path/filepath"
+	"regexp"
 	"sort"
 	"strings"
 )
 
 var (
-	flagOut    = flag.String("o", "", "Output file, else stdout.")
-	flagPkg    = flag.String("pkg", "main", "Package.")
-	flagPrefix = flag.String("prefix", "", "Prefix to strip from filesnames.")
+	flagOut      = flag.String("o", "", "Output file, else stdout.")
+	flagPkg      = flag.String("pkg", "main", "Package.")
+	flagPrefix   = flag.String("prefix", "", "Prefix to strip from filesnames.")
+	flagCondense = flag.Bool("condense", false, "Should we condense the compressed data in the generated source.")
```

Comment:
Yes, change this to just always happen, no option. I'm all for it if it's 2x smaller. We're also getting weird errors in our build server possibly because of the weird characters the compression produces. Base64 may fix that.